tests/BUILD.bazel

```diff
@@ -272,5 +271,43 @@ go_test(
         "//vendor/k8s.io/utils/ptr:go_default_library",
         "//vendor/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1:go_default_library",
         "//vendor/kubevirt.io/qe-tools/pkg/ginkgo-reporters:go_default_library",
-    ],
+    ] + select({
+        "@io_bazel_rules_go//go/platform:386": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:amd64": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:arm": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:arm64": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:mips": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:mips64": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:mips64le": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:mipsle": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:ppc64": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:ppc64le": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:riscv64": [
+            "//pkg/virtctl/usbredir:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:wasm": [
+            "//pkg/virtctl/usbredir:go_default_library",
```

Comment:
Can we not just add for the arches that are supported by kubevirt?

Reply:
I just ran `make generate` and it made it this way. I think same thing had been done by `pkg/virtctl/usbredir`

Reply:
Yeah, this come with s390x result. I was surprised by the verbosity of it as well but I guess with can improve later if needed

https://github.com/kubevirt/kubevirt/commit/49509be6cb238dfcb421aac2b5abf00c60dbf355#diff-f654fc024b6380eba90cf25089c9b77c2ad043a2523a21fac46f7e5aaf031c3bR16